flink-python/pyflink/fn_execution/beam/beam_coder_impl_slow.py

```diff
@@ -186,6 +187,68 @@ def __repr__(self):
         return 'ArrayCoderImpl[%s]' % repr(self._elem_coder)
 
 
+class PickledBytesCoderImpl(StreamCoderImpl):
+
+    def __init__(self):
+        self.field_coder = BinaryCoderImpl()
+
+    def encode_to_stream(self, value, out_stream, nested):
+        coded_data = pickle.dumps(value)
+        real_coded_data = self.field_coder.encode(coded_data)
```

Comment:
Why not call the method `encode_to_stream` of the field_coder? The method of `encode` will create a temporary buffer.